README.md

```diff
@@ -34,10 +34,21 @@ swag init
 Currently all that is located in your .env in the root directory.
 Possible fields are :
 
-| key        | example value                    | description                              |
-|------------|----------------------------------|------------------------------------------|
-|DB_NAME     | test                             | The name of your database                |
-|MONGODB_URI | mongodb://localhost:27017/test   | The connection string to your database   |
-|PORT        | 8080                             | The port to start the server on          |
-|CLIENT_URL  | http://localhost:3000            | The URL the client is requesting from    |
-|BCRYPT_COST | 10                               | The Cost for bcrypt pw generation (>= 10)|
\ No newline at end of file
+| key          | example value                    | description                                            |
+|--------------|----------------------------------|--------------------------------------------------------|
+|DB_NAME       | test                             | The name of your database                              |
+|MONGODB_URI   | mongodb://localhost:27017/test   | The connection string to your database                 |
+|PORT          | 8080                             | The port to start the server on                        |
+|CLIENT_URL    | http://localhost:3000            | The URL the client is requesting from                  |
+|BCRYPT_COST   | 10                               | The Cost for bcrypt pw generation (>= 10)              |
+|PUB_KEY_PATH  | ./auth/public.pem                | The path where the rsa .pem with the public key lives  |
+|PRIV_KEY_PATH | ./auth/private.pem               | The path where the rsa .pem with the private key lives |
+|PRIVATE_PEM_PW| abcdefgh                         | The password your private key .pem is encrypted with   |
+
+To generate the pem files these you may use openssl
+```
+openssl genrsa -des3 -out private.pem 2048
```

Comment:
Why not generate keys on server startup?

Reply:
Also it might be a good idea to add a seperate key-pair so the users password doesnt get sent in cleartext when trying to log in.

Reply:
Makes sense, I like it.
So on server startup we could generate two key pairs, one for password and one for jwt encryption. A user can then fetch the public keys via routes.
I'll get to implementing that asap.